TM1py/Services/CellService.py

```diff
@@ -497,9 +531,10 @@ def execute_unbound_process(self, process: Process, **kwargs) -> Tuple[bool, str
 
         return process_service.execute_process_with_return(process, **kwargs)
 
+    @manage_changeset
     @manage_transaction_log
     def write_values(self, cube_name: str, cellset_as_dict: Dict, dimensions: Iterable[str] = None,
-                     sandbox_name: str = None, **kwargs) -> Response:
+                     sandbox_name: str = None, use_changeset:bool = True, **kwargs) -> Response:
```

Comment:
I understand that to use changesets the user would have to pass `use_changeset` as `True` and a random str value for `changeset` through `kwargs`. Is that correct? 

Perhaps instead of the additional boolean `use_changeset` argument, you could just check if the str `changeset` argument is not None. Do you think that would work?

Reply:
Resolved